src/storage/delta_record.cpp

```diff
@@ -43,16 +59,15 @@ UndoRecord *UndoRecord::InitializeRecord(void *head,
                                          timestamp_t timestamp,
```

Comment:
Not necessary, because this is a static method and will always be called with `UndoRecord::` in front.
If anything I was going to remove `Record` too.

Reply:
That works!